app/views/gov_one/info.html.slim

```diff
@@ -1,14 +1,28 @@
-= render 'learning/cms_debug'
-
 - content_for :page_title do
-  = html_title t('gov_one_info.title')
+  = html_title t('home.title')
+
+- content_for :hero do
+  .govuk-grid-row class='govuk-!-padding-top-9'
+    .govuk-grid-column-three-quarters
+      h1.dfe-heading-xl class='govuk-!-margin-bottom-4'
+        = t('gov_one_info.hero_header')
+      p.govuk-body-l = t('gov_one_info.hero_body')
+
+
+.govuk-grid-row
+  .govuk-grid-column-full
+    . class='govuk-!-margin-bottom-5'
+        = m('gov_one_info.body')
+        hr
+
+    = login_button
 
 .govuk-grid-row
-  .govuk-grid-column-one-half
-    = m('gov_one_info.body')
-
-    - if current_user
-      = logout_button
-    - else
-      = login_button
-  
\ No newline at end of file
+  .govuk-grid-column-full class='govuk-!-margin-top-4 govuk-!-margin-bottom-9'
```

Comment:
Change the containing class to three-quarters

Reply:
[b556ac3](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/968/commits/b556ac3493e853fd89a7adb0e1ba83a10e5608ca)